testdata/asm/gf251_mul.zkasm

```diff
@@ -0,0 +1,57 @@
+;; GF251 curve
+;; Mul on u8, to be able to test all cases
+
+;; u8 version of mul and reduction to be able to test all cases
+
+fn reduction_u8(z u16) -> (RESULT u8) {
+   ;; constants recap
+   ;; R = 256 = 2^8
+   ;; n = 251
+   ;; ninv mod R = 205
+
+   var n u8
+   var ninv_mod_R u8
+
+   var quot u8
+   var tmp_quot u16
+   var quot_times_n u16
+   var discard_bit_shift u8
+   var res u8
+   var red u8
+   var c0, c1 u1
+
+   ninv_mod_R = 205
```

Comment:
Its not good to have registers used to hold constant values (as these create additional columns in the trace).  Better to just use the values directly for now.  At some point, we'll have the ability to create named constants and we can apply them here.  Leave a comment perhaps to clarify though what each constant is.